homeassistant/components/mystrom/models.py

```diff
@@ -0,0 +1,13 @@
+"""Models for the mystrom integration."""
+from dataclasses import dataclass
+
+from pymystrom.bulb import MyStromBulb
+from pymystrom.switch import MyStromSwitch
+
+
+@dataclass
+class MyStromData:
+    """Data class for mystrom device data."""
+
+    device: MyStromSwitch | MyStromBulb
+    info: dict
```

Comment:
Can this be more specific?
```suggestion
    info: dict[str, Any]
```